### PR TITLE
Add a target to find dead links in our documentation.

### PR DESCRIPTION
**Pull request by cyriltovena**
This is not part of the CI because it can be throttle and it's not super fast ~2min.

But still usefull to check that no link are down if we make some big structure changes.

Signed-off-by: Cyril Tovena <cyril.tovena@gmail.com>

### CODE REVIEW

**Comment by timdonovanuk** [comment]
I'm sorry I hurt your feelings <3 and thank you for this :D

File: loki-build-image/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,14 @@ RUN apk add --no-cache curl && \
     mv /tmp/linux-amd64/helm /usr/bin/helm && \
     rm -rf /tmp/linux-amd64 /tmp/helm-$HELM_VER.tgz
 
+FROM alpine as lychee
```
Comment (kavirajk) [inline loki-build-image/Dockerfile:10]: Q: why `lychee?`

**Review by kavirajk — APPROVED** [review]
LGTM!

File: loki-build-image/Dockerfile (hunk 1 above)
Comment (cyriltovena) [reply loki-build-image/Dockerfile:10]: That's the only one I found that is not nodejs and ruby. Which both requires to install the VM to run those and that's just a nono.

**Pull request merged**